aruco_pose/src/aruco_map.cpp

```diff
@@ -339,6 +378,14 @@ class ArucoMap : public nodelet::Nodelet {
 	void addMarker(int id, double length, double x, double y, double z,
 				   double yaw, double pitch, double roll)
 	{
+		// Check whether the id is in range for current dictionary
+		int numMarkers = board_->dictionary->bytesList.rows;
```

Comment:
snake case

Reply:
fixed in https://github.com/CopterExpress/clever/pull/118/commits/dcc5367958c456d21fbf9da26047ba603af7ad5f